OracleTuxedo/core/dockerfiles/12.1.3/Dockerfile

```diff
@@ -7,25 +7,25 @@
 #
 
 # Pull base image
-FROM oraclelinux
+FROM container-registry.oracle.com/java/serverjre
```

Comment:
Do not reference the container registry. You should specify `oracle/serverjre:8` as all the images on GitHub refer to other images that need to be manually built from Dockerfiles provided via this git repo.

Also, please provide a specific version tag, i.e. `oracle/serverjre:8`. Otherwise, you run the risk of this Dockerfile breaking if the major version changes.